lastminutewf2020.yml

```diff
@@ -77,16 +34,28 @@
         name: rsyslog
         state: restarted
 - hosts: all
+  roles:
+    - role: stuvusIT.systemd-timesyncd
+      timesync_timezone: "{{ timezone }}"
+      timesync_ntp_hosts:
+         - "{{ baseip }}.208"
+         - "{{ baseip }}.209"
+         - "{{ baseip }}.210"
+      timesync_fallback_ntp_hosts:
+        - "{{ baseip }}.208"
+        - "{{ baseip }}.209"
+        - "{{ baseip }}.210"
+      tags: systemd-timesyncd
   tasks:
     - name: Update all packages to their latest version
       apt:
         name: "*"
         state: latest
-        only_update: true
+        only_upgrade: true
         update_cache: yes
         cache_valid_time: 3600
         autoremove: yes
-    # TODO boo, this task always runs
+      tags: apt
```

Comment:
Although this is also good, I propose to use the variable `internet` or `blue` in a when to restrict if the replacement for the sources.list should take place. For local testing we can set it to false and when packages is available we can add it there,

Reply:
i added a disable_sources_list_changes for the sources.list changing.